__tests__/app.test.js

```diff
@@ -29,11 +29,38 @@ describe("GET/api/categories", () => {
         });
     });
   });
-
+  describe("GET//api/reviews", ()=>{
+    test('status:200, responds with an array of reviews objects has comment_count keys', () => {
+      return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((response) => {
+          const  reviews  = response.body.reviews;
+          expect(reviews ).toBeInstanceOf(Array);
+          expect(reviews ).toHaveLength(13);
+          reviews.forEach((reviews ) => {
+            expect(reviews ).toEqual(
+              expect.objectContaining({
+                review_id: expect.any(Number),
+                title: expect.any(String),
+                category: expect.any(String),
+                designer: expect.any(String),
+                owner: expect.any(String),
+                review_body: expect.any(String),
+                review_img_url: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                commet_count: expect.any(Number)
+              })
```

Comment:
The ticket asks for the reviews to be sorted by date in descending order, how can you test for this? look into jest-sorted 👍